src/extensions/default/InlineColorEditor/ColorEditor.js

```diff
@@ -64,6 +67,10 @@ define(function (require, exports, module) {
         this._color = tinycolor(color);
         this._originalColor = color;
         this._redoColor = null;
+        this._isUpperCase = PreferencesManager.get("uppercaseColors");
+        PreferencesManager.on("change", "uppercaseColors", function () {
+            this._isUpperCase = PreferencesManager.get("uppercaseColors");
+        }.bind(this));
```

Comment:
Good catch! It sucks that JS objects have a constructor but not a destructor.

@le717 You could add a `ColorEditor.destroy()` method to remove the listener, which could be called from the `InlineColorEditor.onClosed()` method.